apps/src/eval/eval.js

```diff
@@ -151,6 +151,36 @@ Eval.init = function(config) {
     // base's studioApp.resetButtonClick will be called first
     var resetButton = document.getElementById('resetButton');
     dom.addClickTouchEvent(resetButton, Eval.resetButtonClick);
+
+    if (Blockly.contractEditor) {
+      Blockly.contractEditor.registerTestHandlers(function (exampleBlock) {
+        Eval.clearCanvasWithID("test-call");
+        Eval.clearCanvasWithID("test-result");
+        Eval.clearCanvasWithID('user');
```

Comment:
Ah, yes, I'll change the non-`canvg` related references to `svg` throughout the file